src/mean.jl

```diff
@@ -121,7 +120,7 @@ function mean(x::AbstractVector, popsize::AbstractVector, sampsize::AbstractVect
 end
 
 """
-    Survey mean for StratifiedSample objects.
+Estimate the population mean of a variable of a stratified sample, and the corresponding standard error.
     Ref: Cochran (1977)
```

Comment:
Here I would say remove the docstring entirely and include the reference in the comments inside the function body, cause it's only relevant for the implementation, not for the user. What do you say?

Reply:
In the next PR, we'll remove `by` and change `mean` to have a `by` parameter. I'll do that PR right after this one.

Reply:
Ok, but this is not the function for `by`. This is for the mean of a variable from a `StratifiedSample`.